src/linkify-matrix.ts

```diff
@@ -87,6 +93,15 @@ function matrixLinkify(linkify): void {
     S_ROOMALIAS.on(TT.DOT, S_HASH_NAME_COLON_DOMAIN_DOT); // accept repeated TLDs (e.g .org.uk)
     S_ROOMALIAS.on(TT.COLON, S_ROOMALIAS_COLON); // do not accept trailing `:`
     S_ROOMALIAS_COLON.on(TT.NUM, S_ROOMALIAS_COLON_NUM); // but do accept :NUM (port specifier)
+}
+
+function parseMatrixUserId(linkify): void {
+// Text tokens
```

Comment:
```suggestion
    // Text tokens
```

Reply:
Please stop commenting on formatting things if eslint can get it, unless it can't then it needs to be updated. 😄 
It's fixed anyways.